src/Feature/StatusBar/Feature_StatusBar.re

```diff
@@ -40,6 +40,172 @@ module Item = {
   };
 };
 
+module ConfigurationItems = {
+  [@deriving show]
+  type t = {
+    leftItems: list(string),
+    rightItems: list(string),
+    hidden: list(string),
+    showOnNotification: list(string),
+    notificationMode: string,
```

Comment:
It might be nice to have a type for notification mode, like:
```
type notificationMode = 
| Default
| Compact
```

Then, we could add a decoder, like:
```
module Decode = {
   let notificationMode = Json.Decode.(string
   |> map(String.lowercase_ascii)
   |> and_then(fun
   | "default" => succeed(Default)
   | "compact" => succeed(Compact)
   | invalid => fail("Invalid notification mode: " ++ invalid)
   )
};

Reply:
This gets us out of the business of having to check strings elsewhere in the code for this (and will give us a compiler warning if we add a new mode and don't handle it)